src/renderer/components/+config-resource-quotas/resource-quota-details.tsx

```diff
@@ -53,7 +67,7 @@ function renderQuotas(quota: ResourceQuota): JSX.Element[] {
             value={current}
             tooltip={(
               <p>
-                {`Set: ${rawMax}. Usage: ${usage}%`}
+                {`Set: ${rawMax}. Usage: ${+usage.toFixed(2)}%`}
```

Comment:
Is this better? It will show 100 as 100.00? I guess otherwise anything under 1 will show as 1?

Reply:
So this will show `100` as `100` but will correctly show `0.05` as `0.05` instead of what it currently does which is `1`